### PR TITLE
shark registry.json

### PR DESCRIPTION
**Pull request by meta-shark**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1,5 +1,18 @@
 [
-
+   {
+       "ticker": "SHARK",                                                       
+       "poolId": "672f8a27fe8c5115ba67a68d04c81c8604cb68e3d0590d7c085f5a28",  
+       "social": {              
+         "website":"https://medium.com/@metash4rk", 
+         "twitter": "@Meta__Shark",
```
Comment (Bez625) [inline registry.json:7]: Please can you remove the @ from your twitter?

File: registry.json [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+         "facebook": " ",  
+         "instagram": " ",
+         "youtube": " "
+       },
+       "telegram": "metash4rk",                                             
+       "operatorName": "sawhney",  
+       "github": "https://github.com/meta-shark"
```
Comment (Bez625) [inline registry.json:14]: Please can your remove the host from your github? You can just leave this as "meta-shark"

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (meta-shark) [reply registry.json:7]: sure, just saw it, ill get on it. Thanks

File: registry.json (hunk 2 above)
Comment (meta-shark) [reply registry.json]: done !

File: registry.json (hunk 1 above)
Comment (meta-shark) [reply registry.json]: done !

**Pull request merged**